api/src/main/java/org/openmrs/Concept.java

```diff
@@ -542,14 +542,8 @@ public boolean hasName(String name, Locale locale) {
 	public ConceptName getName(Locale locale, ConceptNameType ofType, ConceptNameTag havingTag) {
 		Collection<ConceptName> namesInLocale = getNames(locale);
 		if (!namesInLocale.isEmpty()) {
-			List<ConceptName> matches = new ArrayList<>();
-			
-			for (ConceptName candidate : namesInLocale) {
-				if ((ofType == null || ofType.equals(candidate.getConceptNameType()))
-				        && (havingTag == null || candidate.hasTag(havingTag))) {
-					matches.add(candidate);
-				}
-			}
+			//Pass the possible candidates through a stream and save the ones that match requirements to the list
+			List<ConceptName> matches = new ArrayList<>(namesInLocale.stream().filter(c->(ofType==null || ofType.equals(c.getConceptNameType()))&&(havingTag==null||c.hasTag(havingTag))).collect(Collectors.toList()));
```

Comment:
Can we put some space on the left and right hand side of the operators? Just like the original code was.

Reply:
Updated. Thanks